qoqo_qiskit/tests/backend/test_backend.py

```diff
@@ -438,10 +611,21 @@ def test_run_program() -> None:
     x_circuit += ops.PragmaRepeatedMeasurement("ro_x", 1000, None)
 
     measurement_input = PauliZProductInput(1, False)
-    z_basis_index = measurement_input.add_pauliz_product("ro_z", [0,])
-    x_basis_index = measurement_input.add_pauliz_product("ro_x", [0,])
+    z_basis_index = measurement_input.add_pauliz_product(
+        "ro_z",
+        [
+            0,
+        ],
+    )
+    x_basis_index = measurement_input.add_pauliz_product(
+        "ro_x",
+        [
+            0,
+        ],
+    )
```

Comment:
Is this supposed to be reformatted this way?

Reply:
yep! ruff